src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/projectcreation/model/ProjectCreationModel.kt

```diff
@@ -69,13 +69,22 @@ class ProjectCreationModel {
     }
 
     fun goBack(projectWizard: Wizard) {
-        if (collectionStore.size > 1) {
-            collectionStore.removeAt(collectionStore.size - 1)
-            collectionList.setAll(collectionStore.last().sortedBy { it.sort })
-        } else {
-            projectWizard.back()
+        when {
+            collectionStore.size > 1 -> {
+                collectionStore.removeAt(collectionStore.size - 1)
+                collectionList.setAll(collectionStore.last().sortedBy { it.sort })
+            }
+
+            collectionStore.size == 1 -> {
+                collectionStore.removeAt(0)
+                projectWizard.back()
+            }
+            else -> {
+                projectWizard.back()
+            }
```

Comment:
I believe you can condense this to `else -> projectWizard.back()` since it's only one line